apis/python/tests/test_profiler.py

```diff
@@ -0,0 +1,51 @@
+from tiledbsoma.profiler import profiler, measurements
+
+"""A group of basic tests for the memory and performance profiler decorator"""
+
+
+@profiler(param="tacemalloc")
+def test_tracemalloc():
+    my_list = []
+    for i in range(100000):
+        my_list += [0]
+    return 1
```

Comment:
Can some [assertions](https://docs.pytest.org/en/7.1.x/how-to/assert.html) be added to verify profiler is working as expected?